src/Extracting/DatabaseTransactionHelpers.php

```diff
@@ -30,9 +49,60 @@ private function endDbTransaction()
 
         foreach ($connections as $conn) {
             try {
-                app('db')->connection($conn)->rollBack();
+                tap(
```

Comment:
Can we not use `tap()`? I understand its purpose, but I dislike it because it reads less clearly to me.

Reply:
Yeah, `tap()` and `with()` appear to be relatively marmite in the reception. Consider it swapped out.